src/libutil/util.cc

```diff
@@ -46,6 +46,9 @@ extern char * * environ __attribute__((weak));
 
 namespace nix {
 
+void initLibUtil() {
+}
```

Comment:
This is odd for now, but allows libutil to initialize a dependency if need be in the future.